### PR TITLE
OM-2168 Feature/v1 rfid api

### PR DESCRIPTION
**Pull request by ptuytsch**

### CODE REVIEW

**Comment by svanscho** [comment]
@ptuytsch can we squash some of these commits to make it easier later on to have an oversight or in case of a rollback?

File: src/gateway/rfid_controller.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+@Injectable.named('rfid_controller')
+@Singleton
+class RfidController(object):
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def load_rfid(rfid_id):
+        # type: (int) -> Optional[RfidDTO]
+        rfid_orm = RFID.select().where(RFID.id == rfid_id).first()
```
Comment (LnL7) [inline src/gateway/rfid_controller.py:42]:
```suggestion
        rfid_orm = RFID.get_or_none(id=rfid_id)
```

File: src/gateway/rfid_controller.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            rfid_orm.timestamp_created = RfidController.current_timestamp_to_string_format()
+        if rfid_orm.enter_count is None:
+            rfid_orm.enter_count = 0
+        rfid_orm.save()
+        return RfidMapper.orm_to_dto(rfid_orm)
+
+    @staticmethod
+    def delete_rfid(rfid_id):
+        if RfidController.rfid_id_exists(rfid_id):
+            RFID.delete().where(RFID.id == rfid_id).execute()
```
Comment (LnL7) [inline src/gateway/rfid_controller.py:85]: You can delete unconditionally and check the count returned and then `rfid_id_exists` isn't needed anymore.

File: src/gateway/rfid_controller.py [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+    @staticmethod
+    def current_timestamp_to_string_format():
+        # type: () -> str
+        timestamp = datetime.datetime.now(tzlocal())
+        # replace the microseconds to not show them in the string
+        return timestamp.replace(microsecond=0).isoformat('T')
+
+    @staticmethod
+    def _validate_rfid_type(rfid_dto):
```
Comment (LnL7) [inline src/gateway/rfid_controller.py:103]: unused

**Pull request merged**